### ISSUE TITLE
networks

### ISSUE THREAD

**Issue by transfluxus** [mrzl/ECO#78]
**Text Similarity Measurement using Convolutional Neural Networks**
https://github.com/hohoCode/textSimilarityConvNet

Neural network toolkit for Python http://theanets.rtfd.org
https://github.com/lmjohns3/theanets

**TextualReconstructor**
https://github.com/BarakOshri/TextualReconstructor
Training autoencoders to reconstruct text to generate valuable summarizations of sentences or paragraphs

**neural-storyteller**
https://github.com/ryankiros/neural-storyteller
neural-storyteller is a recurrent neural network that generates little stories about images. This repository contains code for generating stories with your own images, as well as instructions for training new models.

**Deep Encoding and Reconstruction for Normalization of Noisy Text**
Understands slang. only paper
http://noisy-text.github.io/2015/pdf/WNUT23.pdf
no source code

**CNNs for sentence classification**
https://github.com/yoonkim/CNN_sentence

**Convolutional Neural Network for Text Classification in Tensorflow**
https://github.com/dennybritz/cnn-text-classification-tf

**Comment by mrzl** [comment]
i took that list over here, so we can add comments: https://paper.dropbox.com/doc/ECO-research-wbS3Kez47vRaaCLIaHdxI

🆒?

**Comment by transfluxus** [comment]
cool
